x-pack/platform/plugins/shared/lens/public/lens_attribute_service.ts

```diff
@@ -70,11 +69,12 @@ export function getLensAttributeService({
       sharingSavedObjectProps: SharingSavedObjectProps;
       managed: boolean;
     }> => {
-      const { meta, item } = await lensDocumentService.load(savedObjectId);
+      const { item, meta } = await lensDocumentService.load(savedObjectId);
       return {
         attributes: {
-          ...item.attributes,
-          state: item.attributes.state as LensSavedObjectAttributes['state'],
+          ...item,
+          visualizationType: item.visualizationType ?? null,
```

Comment:
In some places the types are misaligned where the schema allows `string | null | undefined` but the runtime value expects `string | null`. This will be cleaned up in a later PR.